book/website/collaboration/inclusive-events/inclusive-events-comms.md

```diff
@@ -0,0 +1,487 @@
+They were also more successful in involving underrepresented people in community events.
+```
+
+(cl-inclusive-events-comms-annoucement)=
+### Announcing your Event 
+After a "Save the Date, you may consider sending out a formal invitation to announce your event after you have finalized details such as the event program and speaker schedule. 
+
+In this announcement, you should provide information about any specific accessibility and inclusion requirements that are addressed such as whether things like captions, interpretation, or other accessibility practices will be available. 
+
+If possible, consider establishing a point of contact on your team who is able to address any questions or concerns that people with lived experience may have about attending your event. 
+This may involve setting up an email inbox such as accessibility@yourevent.com. 
+
+By providing clear and proactive information about how your event addresses barriers to inclusion, people with lived experience will be able to find out whether the event will be accessible to them individually, and decide if they are able to attend. 
+For example, communicating that your event will be hybrid may allow people who otherwise would not engage in the event to attend online. 
+
+This announcement should also contain information about how to register for the event, the finalized location, and a preliminary program. 
+
+```{tip}
+Autism Advocacy suggests that schedules for multi-day events should be sent at least one month in advance which includes details about transport
+and its proximity to the event location, training session names, speaker names, breaks, and planned schedule, and at least 2 weeks in advance for events taking place on a single day.
+The [Autism Advocacy Accessible Event Planning Guide](https://autisticadvocacy.org/wp-content/uploads/2019/05/Accessible-Event-Planning.pdf) also contains other recommendations and timeline guidance for running events for autistic people.
+```
+
+
+(cl-inclusive-events-comms-website)=
+## Using an Event Website 
+It can be incredibly helpful to centralize information about your event in both online and offline locations.
+
+Consider if your event will need a website and dedicate a page on your site to information and a point of contact regarding accessibility requirements and the steps you are taking to make the event inclusive for people with lived conditions and accessibility requirements. 
+
+You can also choose an event platform such as Eventbrite, Humanitix, or EventYay to host information about your event. 
+When deciding on what platform to use please review the accessibility of a potential site, and follow any advice as to how you can make your content on their platform accessible. 
+For example, [Eventbrite provides accessibility guidance](https://www.eventbrite.com/blog/66754-2how-to-creat-an-accessible-event-listing-ds00/) for event pages hosted on their platform. 
```

Comment:
I think there was also the aspect that Eventbrite changed how/what they charge money for, maybe worth to have a quick check if that's still a good recommendation or at least point that those might be paid?

Reply:
Added a small note/disclaimer in https://github.com/the-turing-way/the-turing-way/pull/3327/commits/8d6f2a3453919d6fa2d44c5f3bf92e7bbcc1701a